src/app/components/MostRead/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+  "alpha": true,
```

Comment:
Do we know what this means?

Reply:
I have taken this out now. I noticed it wasn't actually inside the `metadata.json` files for the Healthcheck Component or for VisuallyHiddenText and messaged Dario to confirm. He said it shouldn't be on his screenshot inside the component instructions on storybook and he will update them ASAP. 😄